### DISCUSSION TITLE
Recommendations Trend Summary

### DISCUSSION THREAD

**Discussion by DavidSutherland2** [ranaroussi/yfinance#1307] [Ideas]
Hi,
Great bit of kit, I used to scrape from yahoo into excel using VBA, but since last month Yahoo have encrypted it, and thats when I found yfinance. Time to move my project to the trusty pi!

In my old vba code I used to grab the values from the recommendations summary chart (strong buy, buy, hold, sell, strong sell) that updates monthly, is there functionality to do this in yfinance? I'm assuming it's a summary of the upgrades and downgrades but I can't make them tally when I try to do a similar summary?
Thanks,
David
![recommendations summary](https://user-images.githubusercontent.com/97259715/212567608-66e2a28e-16bc-4ca5-a5f3-61afec6b7169.JPG)

**Comment by kfarokhi** [comment]
Same problem here. In the current version, it seems the "recommendation" method is not coded yet.

[a comment by an automated account is not shown]

**Reply by kfarokhi** [reply]
Please see the stacked bar chart "Analyst Recommendations" under the "Research Analysis" section in the "Analysis" page:
https://finance.yahoo.com/quote/AAPL/analysis?p=AAPL

**Reply by kfarokhi** [reply]
Similarly, the "Upgrades & Downgrades" table at the same location is very informative.

[a comment by an automated account is not shown]
